doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -0,0 +1,192 @@
+To specify the options, the package `Click <https://click.palletsprojects.com>`_
+is used. Another suitable package is the builtin package
+`argparse <https://docs.python.org/3/library/argparse.html>`_.
+
+
+Firstly, the script needs to be converted to a function.
+This can be easily accomplished by using the input arguments
+in a function signature. 
+In this case, the following arguments need to be specified:
+
+* ``n_blades``: Number of blades.
+* ``blade_length``: Length of each blade.
+* ``elastic_modulus``: Elastic modulus of the material.
+* ``density``: Density of the material.
+
+The function is then defined as:
+
+.. literalinclude:: cli_rotor.py
+   :lines: 4-7, 17-22
+
+The value of these parameters are introduced by adding the following code
+right before the function definition:
+
+
+.. literalinclude:: cli_rotor.py
+   :lines: 1-3,8-23
+
+.. warning:: Note that the package *Click* uses decorators (``@click.XXX``), hence
+   it is necessary you specify the *Click* commands right before the function definition.
+
+In addition you need to add the call to the newly created function at the end
+of the script in the following way:
+
+
+.. literalinclude:: cli_rotor.py
+   :lines: 110-
+
+This ensure the new function is called when executing the Python script.
+
+Now you can call your function from the command line using:
+
+.. code:: bash
+
+   $ python rotor.py 4
+   Initializing script with values:
+   Number of blades: 4
+   Blade length: 0.2 m
+   Elastic modulus: 200.0 GPa
+   Density: 7850 Kg/m3
+   Solving...
+   The first natural frequency is 728.57 Hz.
+
+Here ``4`` is the number of blades.
+You can also input other arguments such as:
+
+.. code:: bash
+
+   $ python cli_rotor.py 4 --density 7000
+   Initializing script with values:
+   Number of blades: 4
+   Blade length: 0.2 m
+   Elastic modulus: 200.0 GPa
+   Density: 7000 Kg/m3
+   Solving...
+   The first natural frequency is 771.54 Hz.
+
+
+Advanced usage
+==============
+
+You can use these concepts to make Python create files with specific
+results that can be later used in other apps.
+
+Post-processing images using ImageMagick
+----------------------------------------
+
+For example, you could create an image with PyMAPDL by adding the following
+code to the ``rotor.py`` file:
+
+.. code:: python
+
+   mapdl.vplot(savefig="volumes.jpg")
+
+
+.. image:: volumes.jpg
+
+and use `ImageMagick <https://www.imagemagick.org>`_ to add a frame:
+
+.. code:: bash
+
+   mogrify -mattecolor #f1ce80 -frame 10x10 volumes.jpg
+
+
+and a watermark:
+
+.. code:: bash
+
+   COMPOSITE=/usr/bin/composite
+   $COMPOSITE -gravity SouthEast watermark.jpg volumes.jpg volumes_with_watermark.jpg
+
+where ``-gravity`` is the location of the watermark is case of the watermark is smaller
+than the image, ``COMPOSITE`` is the path to the ImageMagick ``composite`` function,
+``watermark.png`` is the watermark image, and ``volumes_with_watermark.jpg`` is
+the output file.
```

Comment:
```suggestion
Here are descriptions for values used  in the preceding code:

- ``-gravity``: The location of the watermark in case the watermark is
   smaller than the image.
 - ``COMPOSITE``: The path to the ImageMagick ``composite`` function. 
- ``watermark.png``: The name of the PNG file with the watermark image.
- ``volumes_with_watermark.jpg``: The name of the JPG file that the output
   is saved to.
```